src/ansys/dynamicreporting/core/adr_report.py

```diff
@@ -433,7 +433,7 @@ class ReportFetchComponent extends HTMLElement {{
             }}
 
             customElements.define("adr-report", ReportFetchComponent);
-        """
+        """  # noqa

```

Comment:
This is to avoid flake8 trying to import python formatting rules to the JS block